Machine_Learning/smartphone_rate_prediction/main.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import mean_squared_error, r2_score
+
+# Load the dataset
+df = pd.read_csv('C://Users//Deepak//Desktop//dataset.csv')  # Update with your actual file path
```

Comment:
Hey, please provide an example dataset in the directory since it is crucial to test and run the program.
Will review the updated main.py and merge it.
Thank you!